src/assets/wise5/components/match/match-student/match-student-choice-reuse/match-student-choice-reuse.ts

```diff
@@ -15,4 +16,34 @@ export class MatchStudentChoiceReuse extends MatchStudentDefault {
     event.container.element.nativeElement.classList.remove('primary-bg');
     this.studentDataChanged();
   }
+
+  protected addAuthoredChoiceToBucket(choiceId: string, bucket: any): void {
+    bucket.items.push(copy(this.matchService.getChoiceById(choiceId, this.choices)));
+  }
+
+  protected getChoicesThatChangedSinceLastSubmit(latestSubmitComponentState: any): string[] {
+    const choicesThatChanged = super.getChoicesThatChangedSinceLastSubmit(
+      latestSubmitComponentState
+    );
+    const previousBuckets = latestSubmitComponentState.studentData.buckets;
+    for (const currentBucket of this.getNonSourceBuckets()) {
+      const {
+        currentBucketChoiceIds,
+        previousBucketChoiceIds
+      } = this.getPreviousAndCurrentChoiceIds(previousBuckets, currentBucket);
+      const choicesThatWereRemoved = previousBucketChoiceIds.filter(
+        (choiceId: string) => !currentBucketChoiceIds.includes(choiceId)
+      );
+      choicesThatChanged.push(...choicesThatWereRemoved);
+    }
+    return choicesThatChanged;
+  }
+
+  protected checkAnswer(choiceIdsExcludedFromFeedback: string[] = []): void {
+    this.checkAnswerHelper(this.getNonSourceBuckets(), choiceIdsExcludedFromFeedback);
+  }
```

Comment:
Remove need for checkAnswerHelper() by providing function param default (```buckets: any[] = this.getNonSourceBuckets()```)
```suggestion
  protected checkAnswer(
    choiceIdsExcludedFromFeedback: string[] = [],
    buckets: any[] = this.getNonSourceBuckets()
  ): void {
    super.checkAnswer(choiceIdsExcludedFromFeedback, buckets);
  }
```